eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/boot/EventMeshServerTest.java

```diff
@@ -60,56 +60,56 @@ private void testGetConfigWhenStartup(Boolean hasEnv) throws Exception {
+        Assert.assertEquals(22816,config.getGracefulShutdownSleepIntervalInMills());
+        Assert.assertEquals(23816,config.getSleepIntervalInRebalanceRedirectMills());
+        Assert.assertEquals(22816,config.getEventMeshEventSize());
+        Assert.assertEquals(23816,config.getEventMeshEventBatchSize());
     }
 
     private void assertCommonConfig(CommonConfiguration config) {
-        Assert.assertEquals("env-succeed!!!", config.getEventMeshEnv());
-        Assert.assertEquals("idc-succeed!!!", config.getEventMeshIDC());
-        Assert.assertEquals("cluster-succeed!!!", config.getEventMeshCluster());
-        Assert.assertEquals("name-succeed!!!", config.getEventMeshName());
-        Assert.assertEquals("816", config.getSysID());
-        Assert.assertEquals("connector-succeed!!!", config.getEventMeshConnectorPluginType());
-        Assert.assertEquals("storage-succeed!!!", config.getEventMeshStoragePluginType());
-        Assert.assertEquals("security-succeed!!!", config.getEventMeshSecurityPluginType());
-        Assert.assertEquals("registry-succeed!!!", config.getEventMeshRegistryPluginType());
-        Assert.assertEquals("trace-succeed!!!", config.getEventMeshTracePluginType());
-        Assert.assertEquals("hostIp-succeed!!!", config.getEventMeshServerIp());
+        Assert.assertEquals(config.getEventMeshEnv(),"env-succeed!!!");
+        Assert.assertEquals(config.getEventMeshIDC(),"idc-succeed!!!");
+        Assert.assertEquals(config.getEventMeshCluster(),"cluster-succeed!!!");
+        Assert.assertEquals(config.getEventMeshName(),"name-succeed!!!");
+        Assert.assertEquals(config.getSysID(),"816");
+        Assert.assertEquals(config.getEventMeshConnectorPluginType(),"connector-succeed!!!");
+        Assert.assertEquals(config.getEventMeshStoragePluginType(),"storage-succeed!!!");
+        Assert.assertEquals(config.getEventMeshSecurityPluginType(),"security-succeed!!!");
+        Assert.assertEquals(config.getEventMeshRegistryPluginType(),"registry-succeed!!!");
+        Assert.assertEquals(config.getEventMeshTracePluginType(),"trace-succeed!!!");
+        Assert.assertEquals(config.getEventMeshServerIp(),"hostIp-succeed!!!");
 
         List<String> list = new ArrayList<>();
         list.add("metrics-succeed1!!!");
         list.add("metrics-succeed2!!!");
         list.add("metrics-succeed3!!!");
-        Assert.assertEquals(list, config.getEventMeshMetricsPluginType());
+        Assert.assertEquals(config.getEventMeshMetricsPluginType(),list);
 
         Assert.assertTrue(config.isEventMeshServerSecurityEnable());
         Assert.assertTrue(config.isEventMeshServerRegistryEnable());
         Assert.assertTrue(config.isEventMeshServerTraceEnable());
 
-        Assert.assertEquals("eventmesh.idc-succeed!!!", config.getEventMeshWebhookOrigin());
+        Assert.assertEquals(config.getEventMeshWebhookOrigin(),"eventmesh.idc-succeed!!!");
```

Comment:
Please restore the changes in these lines, the original order of 'expect' and 'actual' is correct.